README.md

```diff
@@ -192,6 +195,18 @@ The documentation is being refined and you can contact @JohnGuan via [Discord](h
 
 Help make our documentation better by writing new content for the CasaOS Wiki, correcting existing material, or translating content into new languages.
 
+### Translate CasaOS
```

Comment:
This part should be added to this paragraph

> ### Helping with Translations 🌍 (WIP)

Reply:
Changed